2-environments/envs/development/variables.tf

```diff
@@ -39,3 +39,15 @@ variable "monitoring_workspace_users" {
   description = "Gsuite or Cloud Identity group that have access to Monitoring Workspaces."
   type        = string
 }
+
+variable "project_prefix" {
```

Comment:
@amandakarina how are we plumbing this through to env_baseline? Shouldn't we pass these in here?
https://github.com/amandakarina/terraform-example-foundation/blob/feature/adds-project-prefix-variable/2-environments/envs/development/main.tf

Reply:
Ops! It was missing! Thanks @bharathkkb